src/dvclive/live.py

```diff
@@ -122,7 +122,7 @@ def _init_dvc(self):
                     self._save_dvc_exp = False
                 else:
                     # `DVCLive Only` execution
-                    self._exp_name = random_exp_name(
+                    self._exp_name = get_random_exp_name(
                         self._dvc_repo, self._baseline_rev
```

Comment:
```suggestion
                        self._dvc_repo.scm, self._baseline_rev
```

Or else change the args for `get_random_exp_name`?

Reply:
🙏 Got messed up when resolving conflicts with main